.github/workflows/BUILD.bazel

```diff
@@ -0,0 +1,9 @@
+load("@aspect_bazel_lib//lib:bazelrc_presets.bzl", "write_aspect_bazelrc_presets")
+
+write_aspect_bazelrc_presets(
```

Comment:
Why split `write_aspect_bazelrc_presets` into `.aspect` and `.github`?

Reply:
To support testing with bazel 6 & 7 on CI. See comment above.